earth_enterprise/src/fusion/mtjtools/scroll/application.cpp

```diff
@@ -105,7 +105,7 @@ void ApplicationWindow::loadInitImage(const std::string &image_file_path) {
 void ApplicationWindow::choose()
 {
   QFileDialog *fdlg = new QFileDialog(prevDir, QString::null, this);
-  fdlg->addFilter("Images (*.pre *.img *.tif *.jp2 *.sid *.jpg *.IMG *.TIF *.JP2 *.SID *.JPG)");
+  fdlg->addFilter("Images (*.pre *.img *.tif *.tiff *.jp2 *.sid *.jpg *.IMG *.TIF *.TIFF *.JP2 *.SID *.JPG)");
```

Comment:
Since you're making change here, let'use one pattern for these extensions: lower case then capital case in a consecutive manner.

--> *.pre *.PRE *.img *IMG ....

Instead of having them some time consecutive in some times not consecutive.